pom.xml

```diff
@@ -343,7 +344,7 @@
 		</dependency>
 		<dependency>
 			<groupId>com.databricks</groupId>
-			<artifactId>dbutils-api_2.11</artifactId>
+			<artifactId>dbutils-api_${scala.minor.version}</artifactId>
```

Comment:
This should stay on 2.11, it doesn't exist for 2.12...

Reply:
It stays: `<scala.minor.version>2.11</scala.minor.version>`

Reply:
It should be fixed to scala 2.11. I have another PR where scala.minor.version property will be changed to 2.12 in a maven profile.

Reply:
Done: https://github.com/smart-data-lake/smart-data-lake/pull/13/commits/8e3af5d68bb0e42b54ff6dd8b7db795aa67d48aa